executor_manager/executors/docker/executor.py

```
@@ -68,6 +68,63 @@ def _check_docker_availability(self) -> None:
+        try:
+            result = self.subprocess.run(
+                ["uname", "-r"], capture_output=True, text=True, timeout=5, check=True
+            )
+            kernel_version = result.stdout.strip()
+
+            # Parse kernel version (e.g., "3.10.0-1160.el7.x86_64")
+            version_parts = kernel_version.split(".")
+            if len(version_parts) >= 2:
+                try:
+                    major_version = int(version_parts[0])
+                    minor_version = int(version_parts[1].split("-")[0])
```

Ruff (0.14.13)

Local variable `minor_version` is assigned to but never used

Remove assignment to unused variable `minor_version`

(F841)